City-of-Helsinki/drupal-employment-services
# THF-631: Update guzzlehttp to version 7, update modules

**To test:**
- Checkout branch
- Run `composer install;drush cim -y;drush updb -y`
- Check that there are no pending security updates on: https://drupal-tyollisyyspalvelut-helfi.docker.so/admin/reports/updates
- Check that the TPR integrations work:
  - `drush migrate-status`
  - If any of them are not "Idle", run `drush migrate-reset-status [migration_id]` 
  - Run each migration with `drush migrate-import [migration_id]`
- Click around the site and make sure site sections, content input and the connection to Next.js works

**Updates:**
```
  - Upgrading composer/ca-bundle (1.3.6 => 1.3.7)
  - Upgrading composer/semver (3.3.2 => 3.4.0)
  - Upgrading doctrine/annotations (1.13.3 => 1.14.3)
  - Upgrading doctrine/reflection (1.2.3 => 1.2.4)
  - Upgrading drupal/core (9.5.10 => 9.5.11)
  - Upgrading drupal/core-composer-scaffold (9.5.10 => 9.5.11)
  - Upgrading drupal/core-dev (9.5.10 => 9.5.11)
  - Upgrading drupal/helfi_api_base (2.4.16 => 2.5.5)
  - Downgrading drupal/helfi_azure_fs (1.2.4 => 1.0.3)
  - Upgrading drupal/helfi_tpr (2.2.5 => 2.2.7)
  - Upgrading drupal/matomo (1.21.0 => 1.22.0)
  - Upgrading drupal/oembed_providers (2.1.1 => 2.1.2)
  - Upgrading drupal/paragraphs (1.15.0 => 1.16.0)
  - Upgrading drupal/redirect (1.8.0 => 1.9.0)
  - Upgrading drupal/translatable_menu_link_uri (2.0.0 => 2.1.0)
  - Upgrading drupal/views_bulk_edit (2.8.0 => 2.9.0)
  - Upgrading drupal/views_bulk_operations (4.2.4 => 4.2.5)
  - Upgrading fileeye/mimemap (2.0.1 => 2.0.2)
  - Upgrading guzzlehttp/guzzle (6.5.8 => 7.8.0)
  - Upgrading guzzlehttp/promises (1.5.3 => 2.0.1)
  - Upgrading guzzlehttp/psr7 (1.9.1 => 2.6.1)
  - Upgrading laminas/laminas-feed (2.20.0 => 2.21.0)
  - Upgrading laminas/laminas-stdlib (3.17.0 => 3.18.0)
  - Upgrading league/oauth2-server (8.5.3 => 8.5.4)
  - Upgrading masterminds/html5 (2.7.6 => 2.8.1)
  - Upgrading nikic/php-parser (v4.16.0 => v4.17.1)
  - Upgrading phpdocumentor/type-resolver (1.7.2 => 1.7.3)
  - Upgrading phpstan/phpdoc-parser (1.22.1 => 1.24.1)
  - Upgrading phpunit/php-code-coverage (9.2.26 => 9.2.29)
  - Upgrading phrity/net-uri (1.2.0 => 1.3.0)
  - Upgrading psr/cache (1.0.1 => 3.0.0)
  - Upgrading psr/http-message (1.0.1 => 1.1)
  - Upgrading sebastian/global-state (5.0.5 => 5.0.6)
  - Upgrading sirbrillig/phpcs-variable-analysis (v2.11.16 => v2.11.17)
  - Upgrading slevomat/coding-standard (8.13.1 => 8.13.4)
  - Upgrading steverhoades/oauth2-openid-connect-server (v2.5.0 => v2.6.1)
  - Upgrading symfony/deprecation-contracts (v2.5.2 => v3.3.0)
  - Upgrading symfony/phpunit-bridge (v5.4.25 => v5.4.26)
  - Upgrading symfony/polyfill-ctype (v1.27.0 => v1.28.0)
  - Upgrading symfony/polyfill-iconv (v1.27.0 => v1.28.0)
  - Upgrading symfony/polyfill-intl-idn (v1.27.0 => v1.28.0)
  - Upgrading symfony/polyfill-intl-normalizer (v1.27.0 => v1.28.0)
  - Upgrading symfony/polyfill-mbstring (v1.27.0 => v1.28.0)
  - Upgrading symfony/polyfill-php72 (v1.27.0 => v1.28.0)
  - Upgrading symfony/polyfill-php73 (v1.27.0 => v1.28.0)
  - Upgrading symfony/polyfill-php80 (v1.27.0 => v1.28.0)
  - Upgrading symfony/var-dumper (v5.4.25 => v5.4.28)
````
